example/template.env

```diff
@@ -0,0 +1,29 @@
+# !!! TODO REPLACE WITH ACTUAL RELEASE !!!
+# Image Configuration
+WIKIBASE_IMAGE_NAME=toanwmde/wikibase:1.35.0-wmde.0
```

Comment:
consider moving ports to env variables.

Reply:
Sounds like a good idea… then again, this is just an example, right? Not sure how elaborate we want to make this?

Reply:
Off today's meeting: I'd say we don't want to go overboard, but we do want to offer a decently flexible example config, not just throw complexity at the users and expect them to figure it out. (Which is not an argument to make things too simple, but: external simplicity translates to under-the-hood complexity, which must then be documented)

If env-variablizing the port is too much of an edge case that's fine, but if we call it out in the file we (I) need to mention it in the docs. And if it's worth calling out I thought perhaps it'd be worth turning into a variable.

Reply:
done